R/runNMDS.R

```diff
@@ -42,19 +46,21 @@
 #' @param FUN a \code{function} or \code{character} value with a function
 #'   name returning a \code{\link[stats:dist]{dist}} object
 #'
-#' @param nmdsFUN a \code{character} value to choose the scaling
+#' @param nmds.FUN a \code{character} value to choose the scaling
 #'   implementation, either \dQuote{isoMDS} for
```

Comment:
Can you modify all *.FUN parameters to lowercase nmds.fun? Ley's use only lowercase for parameters to simplify and harmonize